src/routes/extras/containers/ExtrasPage.js

```diff
@@ -37,7 +37,7 @@ export class ExtrasPage extends React.Component<Props> {
   getTileModels (): Array<TileModel> {
     return this.props.extras.map(extra => new TileModel({
       id: extra.alias,
-      name: extra.name,
+      name: extra.title,
```

Comment:
then you could also rename `name` to `title` in the TileModel, right?

Reply:
Sure :D